src/main/java/meditracker/command/AddCommand.java

```diff
@@ -56,6 +56,8 @@ public class AddCommand extends Command {
     private double medicationDosageMorning = 0.0;
     private double medicationDosageAfternoon = 0.0;
     private double medicationDosageEvening = 0.0;
+    private int repeat;
+    private String remarks = "Nil";
```

Comment:
We may want to update the User Guide to state that empty remarks will be stated as "Nil", especially for users who is easily baffled. 
![image](https://github.com/AY2324S2-CS2113T-T09-1/tp/assets/156929749/4d823116-e21b-4dea-8036-6fdc1cf23ba8)

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/210/commits/c233429ef3d492b0911b3cc875afcabe87a79322. Thank you!